Library/Homebrew/test/test_patching.rb

```diff
@@ -122,7 +122,9 @@ def test_single_patch_dsl_with_incorrect_strip
             url PATCH_URL_A
             sha256 PATCH_A_SHA256
           end
-        end.brew(&:patch)
+        end.brew do |f, _staging|
+          f.patch
+        end
```

Comment:
Thanks! Yeah, we still have a lot of work to do to make RuboCop entirely happy with out code, even though we have already disabled or relaxed a lot of its rules in our configuration. :wink: